client/my-sites/sidebar-unified/style.scss

```diff
@@ -119,6 +126,68 @@ $font-size: rem( 14px );
 	.current-site .site,
 	.current-site .all-sites,
 	.current-site__switch-sites {
-		border-color: transparent;
+		border-color: #333333;
+	}
+
+	//client/components/site-selector/style.scss
+	.site-selector .site.is-highlighted {
+		background-color: var ( --color-sidebar-menu-hover-background );
+	}
+
+	.site-selector .site.is-highlighted .site__domain,
+	.site-selector .site.is-highlighted .site__title {
```

Comment:
Worth some basic Sass nesting here?

Reply:
Not sure, this way it is easier to copy over wp-admin and back. @frontdevde any thoughts on this?

Reply:
I guess it depends on whether the intention here is the same as I outlined in `client/assets/stylesheets/_nav-unification.scss`:

```
 * This file contains all relevant styles for the nav unification prototype (see pbAPfg-O2).
 * Using a single place to store these styles will allow us to easily (re)move them later.
 * Depending on the outcome they'll either be deleted or moved to the relevant places.
```

If we're intending to eventually move these styles to replace the styles we're overwriting here, then I'd keep the selectors as is so it'll be easier for our future selves to figure out what we're replacing where.

That said, if these are new styles that should remain in this file then we'd want to use proper nesting.

Reply:
Correct, I have also left as a comment the file `client/components/site-selector/style.scss`